pages/jutsu.php

```diff
@@ -188,10 +188,12 @@ function jutsu(): void {
 
     if($jutsu_to_view == null) {
         $sortedJutsu = [];
-        foreach($player->ninjutsu_ids as $jutsu_id) {
-            $sortedJutsu[] = $player->jutsu[$jutsu_id]->rank;
+        if ($player->ninjutsu_ids) {
+            foreach($player->ninjutsu_ids as $jutsu_id) {
+                $sortedJutsu[] = $player->jutsu[$jutsu_id]->rank;
+            }
+            array_multisort($sortedJutsu, $player->ninjutsu_ids);
         }
-        array_multisort($sortedJutsu, $player->ninjutsu_ids);
     }
```

Comment:
We can actually delete this whole block, not needed

Reply:
Deleted